sdk/security/keyvault/azadmin/backup/autorest.md

```diff
@@ -5,7 +5,7 @@ clear-output-folder: false
 export-clients: true
 go: true
 input-file: 
-    - https://github.com/Azure/azure-rest-api-specs/blob/551275acb80e1f8b39036b79dfc35a8f63b601a7/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.4/backuprestore.json
+    - https://github.com/Azure/azure-rest-api-specs/blob/5dd1107d5f2be8d600325d795450e1d854fbe7e8/specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.5-preview.1/backuprestore.json
```

Comment:
Should be commit a2f6f742d088dcc712e67cb2745d8271eaa370ff. Not sure where this one came from.

Reply:
Commit I'm using came from the issue Pallavi made, changing to the new commit

Reply:
fixed in [9afbf80](https://github.com/Azure/azure-sdk-for-go/pull/21910/commits/9afbf80857d5a559ad6c24a6fddf24714a2a3ee5)